spec/jobs/pusher_job_spec.rb

```diff
@@ -3,8 +3,12 @@
 describe PusherJob, :type => :job do
 
   it "should call twilio with options" do
-    expect(Pusher).to receive(:trigger).with(['channel1', 'channel2'], 'update', {one: 1})
-    PusherJob.new.perform(['channel1', 'channel2'], 'update', {one: 1})
+    channels = ['channel1', 'channel2']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.